app/javascript/lib/vue-components/modules/MultipleLineChart.vue

```diff
@@ -80,7 +80,7 @@ export default {
     }
   },
   mounted() {
-    this.containerChart = document.querySelector('.multiple-line-chart-container');
+    this.containerChart = document.querySelector('.visualizations-home-main');
```

Comment:
Why do you use a non-existent class-name in the component? 🤯

Reply:
This still undone

Reply:
Done

> Some files have not been fixed yet

Done!